doc/specs/#292 - winget should install an app if there is an exact match.md

```diff
@@ -0,0 +1,75 @@
+---
+author: Demitrius Nelon @denelon
+created on: <2020-05-28>
+last updated: <2020-05-28>
+issue id: 292
+---
+
+# Package matching bug fix
+
+## Abstract
+
+The winget.exe client attempts to be generous with the `search` command, but is a bit too generous with `install`. The *id*
+should be the unique key to identifying a package. It should also be case insensitve from the perspective of command execution,
```

Comment:
Version should play a roll in identifying a package. We can default to a specific version where this is concerned, but we should keep it in mind as we move forward.

Reply:
I will call version out.